src/olympia/ratings/views.py

```diff
@@ -78,7 +89,7 @@ class RatingViewSet(AddonChildMixin, ModelViewSet):
     ]
     reply_serializer_class = RatingSerializerReply
     flag_permission_classes = [AllowNotOwner]
-    throttle_classes = (RatingThrottle,)
+    throttle_classes = (RatingThrottle, RatingIPThrottle)
```

Comment:
Similar to my comment above, should we rename `RatingThrottle` to `RatingUserThrottle`?

Reply:
93c84173fb548e4be310032a61d51d878a02c0f8